docs/src/main/asciidoc/cache.adoc

```diff
@@ -597,6 +597,36 @@ public class CacheKeysService {
 }
 ----
 
+=== Populating a `CaffeineCache`
+
+You can populate a `CaffeineCache` using the `CaffeineCache#put(Object, CompletableFuture)` method.
+This method associates the `CompletableFuture` with the given key in the cache. If the cache previously contained a value associated with the key, the old value is replaced by this `CompletableFuture`. If the asynchronous computation fails, the entry will be automatically removed.
```

Comment:
Hmmm, IIUC, it's the result of the `CompletableFuture` that ends up in the cache? With the current wording, it reads as if it was the `CompletableFuture` itself that was stored in the cache.

Reply:
The cache holds the `CompletableFuture`, that's what the AsyncCache does